alignak/scheduler.py

```diff
@@ -119,16 +119,15 @@ def __init__(self, scheduler_daemon):
         self.recurrent_works = {
             0: ('update_downtimes_and_comments', self.update_downtimes_and_comments, 1),
             1: ('schedule', self.schedule, 1),  # just schedule
-            2: ('consume_results', self.consume_results, 1),  # incorporate checks and dependencies
+            2: ('check_freshness', self.check_freshness, 10),
```

Comment:
No, we have discussed this with mohierf and if it's a only passive, not have _active_ check to run, so I have deleted this possibility (if it's only passive, never run the check_command). 

On change status the eventhandler can work ;)

And like before, there is not very different like this, because the consume_result will be called next loop (so less than 1 second) and so not very logical